splunk_add_on_ucc_framework/commands/build.py

```diff
@@ -682,7 +682,7 @@ def generate(
         # clean-up sys.path manipulation
         sys.path.pop(0)
 
-    if global_config:
+    if global_config.has_configuration() or global_config.has_inputs():
```

Comment:
Shall we generate `openapi.json` for add-ons that just have custom rest handlers defined in their globalConfig ([ref doc](https://splunk.github.io/addonfactory-ucc-generator/options/rest_handlers/))? 
Because currently, if there are any developers using it for such a use case, they would no longer get their `openapi.json`.

Reply:
raised a PR to refactor that code first - https://github.com/splunk/addonfactory-ucc-generator/pull/1744